locales/en/app.ftl

```diff
@@ -854,14 +854,16 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 
 -brand-firefox = Firefox
 -brand-monitor = Monitor
--brand-fx-monitor = Firefox Monitor
+-brand-fx-monitor = Mozilla Monitor
 -brand-mozilla = Mozilla
 -brand-premium = Premium
 -brand-monitor-premium = Monitor Premium
```

Comment:
Similarly, looks like we still have 1 lingering reference to "-brand-monitor-premium" which might need to be changed. 
```sh
git grep -P "\{\s?-brand-monitor-premium\s?}" | wc -l # 31
```

And looks like all the usages are for the locales/*/app.ftl `settings-unsubscribe-dialog-confirmation` string:
```sh
git grep -P "\{\s?-brand-monitor-premium\s?}" | grep -v "settings-unsubscribe-dialog-confirmation =" | wc -l # 0
```

Reply:
Hm yeah that's a bit frustrating thanks...